src/views/Pools/components/CardFooter.tsx

```diff
@@ -71,13 +76,23 @@ const Label = styled.div`
 const TokenLink = styled.a`
   font-size: 14px;
   text-decoration: none;
-  color: #12aab5;
+  color: ${(props) => props.theme.colors.primary};
+  cursor: pointer;
+`
+
+const MetamaskIcon = styled.img`
```

Comment:
Use the `Image` component from the uikit, it manages lazy loading http://localhost:6006/?path=/story/components-image--lazy-images